core/templates/dev/head/collection_editor/CollectionEditorNavbarBreadcrumbDirective.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Primary controller for the collection editor navbar breadcrumb.
+ *
+ * @author oskar.cieslik@gmail.com (Oskar Cieslik)
```

Comment:
We don't use ``@author`` tags in JS anymore, since they tend to go out of date and ``git blame`` is the preferred method for finding a contributor who made changes to a file. Please remove this and all other author tags added in this PR.